edb/edgeql/compiler/viewgen.py

```diff
@@ -996,12 +996,13 @@ def get_anchors(stype: s_objtypes.ObjectType) -> RewriteAnchors:
 
             by_type[ty][pn] = (ptr_set, ptrref.real_material_ptr)
 
-    if not anchors:
+    anc = next(iter(anchors.values()), None)
```

Comment:
Is the first one always the one we want?

Reply:
We want any of them, I think it doesn't matter